config/capability_configuration.go

```diff
@@ -56,18 +56,11 @@ func (c CapabilityConfigurations) ToCapabilities(stackId int64) []types.Capabili
 			Name:                cur.Name,
 			ModuleSource:        cur.ModuleSource,
 			ModuleSourceVersion: cur.ModuleSourceVersion,
-			Connections:         types.ConnectionTargets{},
+			Connections:         cur.Connections.Targets(),
 		}
 		if cur.Namespace != nil {
 			capability.Namespace = *cur.Namespace
 		}
-		for key, conn := range cur.Connections {
```

Comment:
Same here, I assume this is because we no longer need to track the stackId on the connections anymore.